greedy_algorithms/activity_selection.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+Problem Statement:
+You are given n activities with their start and finish times. Select the maximum number of activities
+that can be performed by a single person, assuming that a person can only work on a single activity at a time.
+Solution:
```

Comment:
please look at this [example](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) and format your documentation accordingly

Reply:
done